tls/s2n_resume.c

```diff
@@ -57,7 +57,11 @@ static int s2n_tls12_serialize_resumption_state(struct s2n_connection *conn, str
     POSIX_GUARD(conn->config->wall_clock(conn->config->sys_clock_ctx, &now));
 
     /* Write the entry */
-    POSIX_GUARD(s2n_stuffer_write_uint8(to, S2N_TLS12_SERIALIZED_FORMAT_VERSION));
+    if (conn->ems_negotiated && S2N_IN_TEST) {
+        POSIX_GUARD(s2n_stuffer_write_uint8(to, S2N_TLS12_SERIALIZED_FORMAT_VERSION_WITH_EMS));
```

Comment:
Do you want to put a todo / link to an issue to remove the S2N_IN_TEST flag later?